tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -1338,6 +1338,16 @@ public function testBug10772(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug10979(): void
+	{
+		if (PHP_VERSION_ID < 80100) {
+			$this->markTestSkipped('Test requires PHP 8.1.');
+		}
+
+		$errors = $this->runAnalyse(__DIR__ . '/data/bug-10979.php');
+		$this->assertCount(80, $errors);
```

Comment:
Can you please fix the example so that it's valid and does not report any errors? And make sure it still reproduces the performance problem.

I'm worrying that the number 80 is going to fluctuate and is going to fail quite often as we work on PHPStan.

Reply:
we now have a zero errors repro